invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -446,8 +446,27 @@ dl.details-list {
   }
 }
 
+.ui.header.request-header {
+  .creatibutors {
```

Comment:
This can be removed if you move the `<dl>` out of `<h2>`.

Reply:
done, thanks for the comments :+1:

Reply:
Looks good, thanks! :)